embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/payload/Session.kt

```diff
@@ -142,6 +141,11 @@ internal data class Session @JvmOverloads internal constructor(
 
     companion object {
 
+        /**
+         * Signals to the API that the application was in the foreground.
+         */
+        internal const val APPLICATION_STATE_FOREGROUND = "foreground"
+
         @JvmStatic
         fun buildStartSession(
```

Comment:
Do we even need this given that we're getting rid of the startSession message, or is this doing something else as well?

Reply:
We still need this right now as the start message is sent by default. After it's removed we will still need it to populate initial session state, but can rename it to something more appropriate